integration-tests/src/test/kotlin/com/google/devtools/ksp/test/DependencyResolutionCheckIT.kt

```diff
@@ -23,19 +23,16 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
-import java.io.File
 
 @RunWith(Parameterized::class)
-class AGP741IT(useKSP2: Boolean) {
+class DependencyResolutionCheckIT(useKSP2: Boolean) {
     @Rule
     @JvmField
     val project: TemporaryTestProject = TemporaryTestProject("playground-android-multi", "playground", useKSP2)
 
     @Test
     fun testDependencyResolutionCheck() {
-        val gradleRunner = GradleRunner.create().withProjectDir(project.root).withGradleVersion("7.6.3")
-
-        File(project.root, "gradle.properties").appendText("\nagpVersion=7.4.1")
```

Comment:
Can we keep this? We need to cover users of older Gradle and AGP versions.

Reply:
I only changed this because the CI test was failing with the following error:

```
Could not determine the dependencies of task ':workload:compileDebugKotlin'.
    > Failed to install the following SDK components:
          tools Android SDK Tools
```

It was passing for me locally, so I wasn't sure how to troubleshoot.

This test isn't really specific to AGP 7.4.1... it will fail similarly with AGP 8.7.0 if resolving dependencies early.

FWIW, there is still AGP731IT which is a similar test using AGP 7.3.1 and the same Gradle version.

But the bottom line is that I'm happy to leave this test as-is, but I don't know how to troubleshoot the test failure :)